requirements.txt

```diff
@@ -1,4 +1,4 @@
-pyquil>=2.4.0
+pyquil>=2.7.0
```

Comment:
This change must imply a major version bump. We need to either revert the change to pyquil and re-release, or re-label this release as 3.0.0.